.ci/build.yml

```diff
@@ -17,6 +17,7 @@ parameters:
   publishOutputSuffix: ''                                   # the artifact suffix to use when publishing the output folder
   signListPath: 'SignList.xml'                              # the path to the SignList.xml to copy into the nuget artifact for signing
   # job software version parameters
+  agentPool: 'Azure Pipelines'                              # the name of the Azure VM pool
```

Comment:
Maybe we should rename this `agentPoolName` to improve clarity on the callers side.